lab-jaren/lib/read-many-files.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+const async = require('async');
```

Comment:
You did some serious google-fu to find this cool package. Awesome job at being so resourceful.